src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/snippets_mark_skipped.cpp

```diff
@@ -0,0 +1,285 @@
+bool isSuitableMatMulParent(const std::shared_ptr<const Node> &node) {
+    const bool is_suitable_node = ov::is_type<ov::op::v0::MatMul>(node);
+    // has a single output, connected to a single child
+    const auto out = node->outputs();
+    const bool has_only_child = (out.size() == 1) && (out[0].get_target_inputs().size() == 1);
+    return is_suitable_node && has_only_child;
+}
+bool isSuitablePoolChild(const std::shared_ptr<const Node> &node) {
+    const bool is_suitable_node = ov::is_type<ov::op::v1::MaxPool>(node);
+    // has a single output, connected to a single child
+    const auto out = node->outputs();
+    const bool has_only_child = (out.size() == 1) && (out[0].get_target_inputs().size() == 1);
+    return is_suitable_node && has_only_child;
+}
+bool isSuitableChildForFusingSimple(const std::shared_ptr<const Node> &node) {
+    // Note: Fusing child is allowed to have several users, but that must be the end of the chain
+    return SupportsFusingWithConvolution_Simple(node) && getNumNonConstInputs(node) == 1;
+}
+bool isSuitableChildForFusingBias(const std::shared_ptr<const Node> &node, int fusingAxis) {
+    auto is_suitable_node = [](const std::shared_ptr<const Node> &node) {
+        return  (ov::is_type<ov::op::v1::Convolution>(node) ||
+                 ov::is_type<ov::op::v1::GroupConvolution>(node) ||
+                 ov::is_type<ov::op::v0::MatMul>(node)) &&
+                 node->inputs().size() == 2;
+    };
+    const bool is_bias = ov::is_type<ov::op::v1::Add>(node);
+    if (!is_bias)
+        return false;
+    const auto in = node->inputs();
+    if (in.size() != 2)
+        return false;
+    const bool has_suitable_parent = is_suitable_node(in[0].get_source_output().get_node_shared_ptr()) ||
+                                     is_suitable_node(in[1].get_source_output().get_node_shared_ptr());
+    if (!has_suitable_parent)
+        return false;
+    for (const auto &in : node->inputs()) {
+        const auto& parent_out = in.get_source_output();
+        size_t bias_port = 1 - in.get_index();
+        const auto& bias_out = node->input_value(bias_port);
+        const auto bias_node = bias_out.get_node_shared_ptr();
+        const bool is_input_type = ov::is_type<ov::op::v0::Parameter>(bias_node) ||
+                                   ov::is_type<ov::op::v0::Constant>(bias_node) ||
+                                   ov::is_type<ov::op::v0::Result>(bias_node) ||
+                                   ov::is_type<ov::op::v3::ReadValue>(bias_node) ||
+                                   ov::is_type<ov::op::v6::ReadValue>(bias_node);
+        if (!is_input_type)
+            continue;
+        if (bias_node->outputs().size() != 1)
+            continue;
+        const auto& parent_pshape = parent_out.get_partial_shape();
+        const auto& bias_pshape = bias_out.get_partial_shape();
+        if (parent_pshape.is_dynamic() || bias_pshape.is_dynamic())
```

Comment:
I think we can make it easier 😃

Could you take a look please?

```
bool isSuitableChildForFusingBias(const std::shared_ptr<const Node> &node, int fusingAxis) {
    if (!ov::is_type<ov::opset1::Add>(node))
        return false;

    auto is_suitable_parent= [](const std::shared_ptr<const Node> &node) {
        return  (ov::is_type<ov::op::v1::Convolution>(node) ||
                 ov::is_type<ov::op::v1::GroupConvolution>(node) ||
                 ov::is_type<ov::op::v0::MatMul>(node));
    };

    for (const auto &in : node->inputs()) {
        const auto& parent_out = in.get_source_output();
        const auto& parent = parent_out.get_node_shared_ptr();
        const auto& parent_pshape = parent_out.get_partial_shape();
        if (is_suitable_parent(parent) && parent_pshape.rank().is_static()) {
            if (parent->get_output_target_inputs(0).size() > 1)
                break;
            const auto bias_port = 1 - in.get_index();
            const auto bias_out = node->input_value(bias_port);
            if ((bias_out.get_target_inputs().size() > 1) || !ov::op::util::is_on_constant_path(bias_out))
                break;
            const auto& bias_pshape = bias_out.get_partial_shape();
            if (bias_pshape.is_dynamic())
                break;
            const auto bias_shape_norm = getNormalizedDimsBySize(bias_pshape.get_shape(), parent_pshape.size());
            if (fusingAxis >= static_cast<int>(bias_shape_norm.size()) || fusingAxis >= static_cast<int>(parent_pshape.size()) ||
                bias_shape_norm.size() != parent_pshape.size() || bias_shape_norm.size() < 2)
                break;
            if (parent_pshape[fusingAxis].is_dynamic())
                break;
            if (((bias_shape_norm[fusingAxis] == parent_pshape[fusingAxis].get_length())) &&
                (bias_shape_norm[fusingAxis] == static_cast<int64_t>(shape_size(bias_shape_norm))))
                return true;
        }
    }
    return false;
}
```

What do you think?

I provides this solution based on the following [conditions](https://github.com/openvinotoolkit/openvino/blob/ee2aedf910a07dc087e508391d250263fa5db3c1/src/plugins/intel_cpu/src/graph_optimizer.cpp#L556):
```
        const auto channelAxis = parentNode->getFusingAxis();
        if (!dimsEqualStrong(biasDims[channelAxis], parentOutDims[channelAxis]))
            return false;

        for (size_t i = 0; i < biasDims.size(); i++) {
            if (biasDims[i] != 1 && static_cast<int>(i) != channelAxis)
                return false;
        }
```

It means that `Bias` must have static shapes

Reply:
Thanks Alexandra for comment! I've tried this version, the unit tests for fusing bias doesn't pass with the following errors. I'm trying the root cause the issue.
smoke_Snippets_SkipConvFused_ConvBiasRelu: "`Subgraph/SnippetsOpset != Relu/opset1`"
smoke_Snippets_SkipConvFused_ConvBiasTwoRelu: "`different SubGraph InputDescription`"
smoke_Snippets_SkipMatMulFused_MatMulBiasTwoRelu: "`Subgraph/SnippetsOpset != Relu/opset1`"
smoke_Snippets_SkipMatMulFused_MatMulBiasReluDiv: "`different SubGraph InputDescription`"

Reply:
Cool suggestion! Applied. The errors mentioned above is on test case. Bias fusing needs test cases to feed Const input, not Parameter. I've updated the test cases accordingly. Thanks Alexandra!